source/ai-data-usage.txt

```diff
@@ -23,14 +23,22 @@ The |copilot| is powered by `GitHub Copilot
 How Your Data is Used
 ---------------------
 
-When you use the |copilot| , the following information is sent to
-MongoDB's backend, the MongoDB Docs Chatbot, and/or the third party AI
-provider: 
+MongoDB stores the following information:
 
-- The full text of your natural language prompt.
-- The schema of the collection you are using, 
-  including database name, collection name, field names, and types. 
+- The full text of your natural language prompt
+- The full text of the generated response
+- The schema of the collection you are using including, but not limited
+  to, database name, collection name, field names, and types
 
-The information that is sent will not be shared with any other third 
-parties or stored by the AI provider. We do not send database 
-connection strings, credentials, or rows/documents from your databases. 
+GitHub Copilot stores the following information:
+
+- The full text of your natural language prompt
+- The full text of the Copilot generated response
+- User engagement and feedback data 
+
+The information that is sent will not be shared with any other third
+parties. We do not send out database connection strings, credentials, or
+rows/documents from your databases. 
```

Comment:
If the user enabled the setting for `Enable sending sample field values with the VSCode copilot chat @MongoDB participant /query command` then documents could make their way into the prompts. Should we call that out?

Reply:
I added a mention of this setting!